### DISCUSSION TITLE
Actually collection directory is not deleting from S3 bucket when deleting collection

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#36655] [Q&A and General discussion]
Hi team,
When we are deleting collections using attu. the actually data dir for that collection in S3 bucket if not deleting. Could you please help me to find out the reason?

**Comment by navneetrajput06** [comment]
@yhmo

**Comment by xiaofan-luan** [comment]
It takes some time to do GC. 
Is has some protection period and also gc is backend task.
Please wait for one day and check

**Reply by navneetrajput06** [reply]
That still present for all previous collection ever those were deleted a month back.

**Reply by yhmo** [reply]
@navneetrajput06 
The directories of the deleted collections(deleted a month ago) are empty, right?

**Reply by xiaofan-luan** [reply]
> That still present for all previous collection ever those were deleted a month back.

Which milvus version are u using? if you are using 2.3.22 or 2.4.x, please share the log.
or you need to check why datacoord is not doing gc.
